smart-contracts/contracts/mixins/MixinGrantKeys.sol

```diff
@@ -23,8 +23,8 @@ contract MixinGrantKeys is
     uint[] calldata _expirationTimestamps,
     address[] calldata _keyManagers
   ) external
-    onlyKeyGranter
```

Comment:
Shouldn't we fix the modifier instead?

Reply:
Looking again, I think that's a better approach. I was thinking that There were some case where KeyGranter should be allowed access but not LockManager, but this isn't the case.